includes/admin/class-wp-job-manager-promoted-jobs-admin.php

```diff
@@ -114,6 +128,55 @@ public function add_action_notice( $actions_handled ) {
 		return $actions_handled;
 	}
 
+	/**
+	 * Add the host of the Promote Job form to the array of allowed redirect hosts.
+	 *
+	 * @param array $hosts Allowed redirect hosts.
+	 * @return array Updated array of allowed redirect hosts.
+	 */
+	public function add_to_allowed_redirect_hosts( $hosts ) {
+		$hosts[] = wp_parse_url( WP_Job_Manager_Helper_API::get_wpjmcom_url(), PHP_URL_HOST );
```

Comment:
I imagine you're using `wp_parse_url ` for the case that the user is using the filter `wpjm_wpjmcom_api_url`, right? I wonder if it would make more sense in the `get_wpjmcom_url` method.

Reply:
Actually, the idea is just to return the host involved, because that's what I want to get here. `get_wpjmcom_url` returns a full URL, with protocol and even a path if needed (one can technically run a dev version of WPJMCOM on `http://localhost/wpjobmanager.com`, maybe?), meanwhile here I just want to get the host, and nothing else, hence why I'm using `wp_parse_url`.

Does that make sense?

Reply:
Oh! Sorry! I misunderstood it while reading previously! That makes sense!